internal/tui/model_list.go

```diff
@@ -118,31 +182,38 @@ func (m ListModel) KeyMap() *KeyMap {
 	))
 	kmap.Set("up", m.list.KeyMap.CursorUp)
 	kmap.Set("down", m.list.KeyMap.CursorDown)
+	kmap.Set("t", key.NewBinding(
+		key.WithKeys("t"),
+	))
+	kmap.Set("e", key.NewBinding(
+		key.WithKeys("e"),
+	))
 
 	return kmap
 }
 
-func (m ListModel) Init() tea.Cmd {
-	return tea.Batch(m.startSearch, m.spinner.Tick)
-}
-
-type suggestionsMsg struct {
-	suggestions []string
-}
+func editInputKeys() *KeyMap {
```

Comment:
```suggestion
func (m ListModel) EditKeyMap() *KeyMap {
```

To be consistent with the other key map implementation

This removes the need for the `editKeys` field